Lesson-004/Lesson-004.R

```diff
@@ -5,7 +5,7 @@ library(dplyr)
 # Load the file Lesson-004/data/cervix.csv and store it in a variable called 
 # "cervix": 
 
-
+cervix<-read.csv("Lesson-004/data/cervix.csv")
```

Comment:
I had been leaving the `read.csv` line blank to make them fill it in as a reminder of how to do it. Otherwise great!